### DISCUSSION TITLE
Starting a REPL with an alternate Lisp core

### DISCUSSION THREAD

**Discussion by pamoroso** [lem-project/lem#1201] [Q&A] [answered]
When starting a REPL with `C-c C-c` (`M-x start-lisp-repl`) is it possible to interactively select a Lisp core file other than the default? I mean the equivalent of SBCL's `--core` command line option.

A use case I have in mind is starting a SBCL REPL with McCLIM preloaded. Loading McCLIM in SBCL takes a few dozen seconds. So I'm wondering whether it's possible to interactively tell Lem to start SBCL with a core containing McCLIM instead of the default core.

**Comment by cxxxr** [comment, marked as answer]
How about the command `C-u M-x slime`?

**Comment by pamoroso** [comment]
This is what I was looking for, thanks. I need to experiment a bit as launching in Lem a saved McCLIM core yelds an error.
